checks/raw/sast.go

```diff
@@ -67,6 +67,11 @@ func SAST(c *checker.CheckRequest) (checker.SASTData, error) {
 	}
 	data.Workflows = append(data.Workflows, sonarWorkflows...)
 
+	snykWorkflows, err := getSnykWorkflows(c)
```

Comment:
i'll file an issue to de-dupe this. no need to have the same function 3 times, but it's outside of the scope of this PR.

Reply:
Sounds good, if you create an issue then I'll fix the issue after this PR has landed